### DISCUSSION TITLE
3.1.0-beta.1

### DISCUSSION THREAD

**Discussion by janusw** [dotnet-bluetooth-le/dotnet-bluetooth-le#779] [Announcements]
- #746 Windows.Devices.Bluetooth Only
- #764 ReadAsync updates characteristic value on Windows
- #770 Remove .NET 6
- #773 Added BroadcastName to AdvertisementRecordType (fixes #772)
- #774 disable MainThreadInvoker in Adapter.ConnectToDeviceAsync (fixes #757)
- #776 Improve Connect In Windows
- #777 StateChanged for Windows
- #778 Add support for .NET 8

<hr /><em>This discussion was created from the release <a href='https://github.com/dotnet-bluetooth-le/dotnet-bluetooth-le/releases/tag/3.1.0-beta.1'>3.1.0-beta.1</a>.</em>

**Comment by axa88** [comment]
Introduces issue #828
